src/GeneralStateTestsFiller/stCreateTest/CreateMaxCodesizeSuicideFiller.yml

```diff
@@ -0,0 +1,79 @@
+# Use CREATE opcode to create a contract with a codesize equal to the maximum allowed (0x6000) then destroy the contract
+CreateMaxCodesizeSuicide:
+
+  env:
+    currentCoinbase: 2adc25665018aa1fe0e6bc666dac8fc2697ff9ba
+    currentDifficulty: '0x20000'
+    currentGasLimit: 60000000
+    currentNumber: "1"
+    currentTimestamp: "1000"
+    previousHash: 5e20a0453cecd065ea59c37ac63e079ee08998b6045136a8ce6635c7912ec0b6
+
+  pre:
+    00000000000000000000000000000000000000d0:
+      balance: 0
+      code: ''
+      nonce: '0'
+      storage: {}
+
+    00000000000000000000000000000000deadbeef:
+      balance: 2
+      code: |
+        :yul
+        {
+          // initcode: { push2(0x6000), dup1, push1(0x0c), push1(0x00), codecopy, push1(0x00), return,
```

Comment:
need an example where it does not suicide. to see that this code does actually creates a contract with max allowed codesize

Reply:
In Andrew's tests it was very clear. here is a bit hard to read